mainapp/models.py

```diff
@@ -407,3 +407,13 @@ class Meta:
 
     def __str__(self):
         return self.description[:100]
+
+class ParsedSMSData(models.Model):
+    parsed_count = models.IntegerField(null=True,blank=True,default=0,)
```

Comment:
since there's a lot of repetition, you can declare the attributes in a dict like
```
attribute_properties = {'null': True, 'blank': True, 'default': 0}
```

and use it as `**attribute_properties`.